cv.typ

```diff
@@ -0,0 +1,70 @@
+#show heading: set text(font: "Linux Biolinum")
+
+#show link: underline
+#set page(
+ margin: (x: 0.9cm, y: 1.3cm),
+)
+#set par(justify: true)
+
+#let chiline() = {v(-3pt); line(length: 100%); v(-5pt)}
+
+= Eva Pace (she/her)
+
+#link("https://github.com/evaporei")[Github] | #link("https://gitlab.freedesktop.org/eva")[Gitlab] | #link("https://www.linkedin.com/in/evapace/")[LinkedIn] |
+#link("mailto:eba.pachi@gmail.com")[eba.pachi\@gmail.com]
+
+#chiline()
+
+Software Engineer with 6 years of experience. Worked with Web, FFI, WebAssembly and Blockchain. Currently looking for a low-level role in compilers or computer graphics.
+
+== Experience
+#chiline()
+
+*Rust Engineer* #h(1fr) 2021/04 -- 2023/02 \
+Edge & Node #h(1fr) San Francisco, US (remote)
+\
+I've worked to make #link("https://thegraph.com")[The Graph] move forward in the decentralization of public data. Delivered features such as: \
+- Integration of our first non-EVM compatible chain, NEAR Protocol;
```

Comment:
Try to avoid `our` or `I` and use verbs directly with the actor assumed to be you. Similar idea as the following bullet points. 
For example, here you can have something like `Integrated the first non-EVM compatible chain ([NEAR Protocol](...)) into the client.`...?